src/book/models/InvestmentAccount.ts

```diff
@@ -42,9 +42,6 @@ export default class InvestmentAccount {
       this.account.commodity.mnemonic,
       DateTime.now(),
     );
-    if (price.guid === 'missing_price') {
-      throw new Error(`No price found for ${this.account.commodity.mnemonic}`);
-    }
```

Comment:
this is because when it's in currency, we will returning missing_price with value 1.